Framework/API/inc/MantidAPI/MatrixWorkspace.h

```diff
@@ -512,7 +512,7 @@ class MANTID_API_DLL MatrixWorkspace : public IMDWorkspace,
   /// Flag indicating whether the data has common bins. False by default
   mutable bool m_isCommonBinsFlag;
 
-  /// The set of masked bins in a map keyed on spectrum index
+  /// The set of masked bins in a map keyed on spectrum number
```

Comment:
I know is this only a comment but I think this might cause confusion as I think the key is the 0-based vector index.